src/openvino-ir.js

```diff
@@ -0,0 +1,656 @@
+            if (schema.description) {
+                schema.description = marked(schema.description);
+            }
+            if (schema.attributes) {
+                schema.attributes.forEach((attribute) => {
+                    if (attribute.description) {
+                        attribute.description = marked(attribute.description);
+                    }
+                });
+            }
+            if (schema.inputs) {
+                schema.inputs.forEach((input) => {
+                    if (input.description) {
+                        input.description = marked(input.description);
+                    }
+                });
+            }
+            if (schema.outputs) {
+                schema.outputs.forEach((output) => {
+                    if (output.description) {
+                        output.description = marked(output.description);
+                    }
+                });
+            }
+            if (schema.references) {
+                schema.references.forEach((reference) => {
+                    if (reference) {
+                        reference.description = marked(reference.description);
+                    }
+                });
+            }
+            return schema;
+        }
+        return '';
+    }
+
+    getInputs(type, inputs) {
+        var results = [];
+        var index = 0;
+
+        inputs.slice(index).forEach((input) => {
+            const name = (index === 0) ? 'input' : ('(' + index.toString() + ')');
+            results.push({
+                name: name,
+                connections: [{id: input}]
+            });
+            index++;
+        });
+
+        return results;
+    }
+
+    getOutputs(type, outputs, layerName) {
+        var results = [];
+        var index = 0;
+
+        outputs.slice(index).forEach((output) => {
+            const name = (index === 0) ? 'output' : ('(' + index.toString() + ')');
+            results.push({
+                name: name,
+                connections: [{id: layerName}]
+            });
+            index++;
+        });
+        return results;
+    }
+
+    getAttributeVisible(operator, name, value) {
+        var schema = this._map[operator];
+        if (schema && schema.attributes && schema.attributes.length > 0) {
+            if (!schema.attributesMap) {
+                schema.attributesMap = {};
+                schema.attributes.forEach((attribute) => {
+                    schema.attributesMap[attribute.name] = attribute;
+                });
+            }
+            var attribute = schema.attributesMap[name];
+            if (attribute) {
+                if (attribute.hasOwnProperty('visible')) {
+                    return attribute.visible;
+                }
+                if (attribute.hasOwnProperty('default')) {
+                    return value != attribute.default.toString();
+                }
+            }
+        }
+        return true;
+    }
+}
+
+openvinoIR.DotModel = class {
```

Comment:
`openvinoDot.Model`

Reply:
@lutzroeder I made renaming and it seems already enough, not needed to make the sub namespace. 

is the discussion resolved now?